api/src/API/Controllers/WildApricotController.cs

```diff
@@ -0,0 +1,119 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using RaceResults.Common.Models;
+using RaceResults.Data.KeyVault;
+using RestSharp;
+
+namespace RaceResults.Api.Controllers
+{
+    [ApiController]
+    [Route("wa")]
+    public class WildApricotController : ControllerBase
+    {
+        public static string Base64Encode(string plainText)
+        {
+            var plainTextBytes = System.Text.Encoding.UTF8.GetBytes(plainText);
+            return System.Convert.ToBase64String(plainTextBytes);
+        }
+
+        // FOR TESTING. Need to figure out something better...
+        public static bool OverrideAuth { get; set; }
+
+        public static async Task<bool> Authorized(HttpRequest request, string orgAssignedMemberId)
```

Comment:
The way to avoid having this override is to leverage dependency injection and avoid using static classes/methods. We can then inject an implementation that always returns authorizing when testing and one that actually does validation when running.

Reply:
Do you think you could do this refactoring?